.travis.yml

```diff
@@ -24,12 +24,16 @@ branches:
     - /^amp-release-.*$/
 addons:
   apt:
+    sources:
+      - ubuntu-toolchain-r-test
     packages:
       # Use unbuffer (from expect-dev) for log coloring (github.com/travis-ci/travis-ci/issues/7967)
       - expect-dev
       - google-cloud-sdk
       - openssl
       - protobuf-compiler
+      - wget
```

Comment:
Travis VMs come with `wget` (we already use it for the sauce proxy), so this line is unnecessary.